presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -397,6 +397,19 @@ public void testMapInSubquery()
                 "SELECT * FROM VALUES (3,-3)");
     }
 
+    // TODO: for now we forbid all indeterminate values in the probe side, this test should be removed after indeterminate values are allowed in probe side
+    @Test
+    public void testInvalidIndeterminateInSubquery()
+            throws Exception
+    {
+        assertQueryFails("select x from (values null, 1) t(x) where x in (values 1, 2, 3, null)",
+                "indeterminate values are not allowed in probe side");
+        assertQueryFails("select x from (values null, 1) t(x) where (x in (select * from (values 1) p(y) where y < 0)) is null",
+                "indeterminate values are not allowed in probe side");
+        assertQueryFails("select x from (values map(array[1], array[cast(null as integer)])) t(x) where x in (values map(array[1], array[1]))",
+                "indeterminate values are not allowed in probe side");
```

Comment:
Add also simple cases (both should fail)
```
select array[3,cast(null as integer)] in (values array[2,2]);
select array[2,2] in (values array[3,cast(null as integer)]);
```

Also, please add cases using maps and `row`